cmd/influxd/inspect/verify_tsm/verify_tsm_test.go

```diff
@@ -18,6 +18,7 @@ func TestInvalidChecksum(t *testing.T) {
 	verify := NewTSMVerifyCommand()
 	b := bytes.NewBufferString("")
 	verify.SetOut(b)
```

Comment:
Why this change?

Reply:
For whatever reason, when upgrading Cobra and Viper, it seems like some semantics changed as to where the output ends up (err or out), and it was causing test failures. So I went ahead and did this for all test files that had any errors just to make sure they won't fail (just a couple inspect command tests)